test/unit/spec/signaling/v2/peerconnection.js

```diff
@@ -1975,6 +2007,9 @@ function makeDescription(type, options) {
       type === 'pranswer') {
     const session = 'session' in options ? options.session : Number.parseInt(Math.random() * 1000);
     description.sdp = 'o=- ' + session + '\r\n';
+    if (options.iceLite) {
+      description.sdp += 'a=ice-lite\r\n';
+    }
```

Comment:
What if the agent is not ice-lite, should the mocked sdp indicate trickle-ice support?

Reply:
@ceaglest 

`a=ice-lite` is a session attribute, where as `a=ice-options:trickle` is an m= section specific attribute. So, I think we can safely assume that if `a=ice-lite` is not present, then the peer either supports trickle-ice or it gathers all candidates beforehand.